tests/robustness/main_test.go

```diff
@@ -0,0 +1,55 @@
+// +build darwin linux
+
+package robustness
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os"
+	"testing"
+
+	engine "github.com/kopia/kopia/tests/robustness/test_engine"
```

Comment:
out of curiosity, why not just rename the package to avoid aliasing?

Reply:
Good point. I don't recall having any specific reason for doing it this way. I'll address this in a future PR.